content/news/2024-02-17-bevy-0.13/index.md

```diff
@@ -0,0 +1,3215 @@
+
+We've swapped the material and mesh bind groups, so that mesh data is now in bind group 1, and material data is in bind group 2. This greatly improved our draw call batching when combined with changing the sorting functions for the opaque passes to sort by pipeline and mesh. Previously we were sorting them by distance from the camera. These batching improvements mean we're doing fewer draw calls, which improves CPU performance, especially in larger scenes. We've also removed the `get_instance_index` function in shaders, as it was only required to workaround an upstream bug that has been fixed in wgpu 0.19. For other shader or rendering changes, please see the [migration guide](/learn/migration-guides/0.12-0.13/) and [wgpu's changelog](https://github.com/gfx-rs/wgpu/blob/v0.19/CHANGELOG.md).
+
+Many small changes both to Bevy and `wgpu` summed up to make a modest but measurable difference in our performance on realistic 3D scenes!
+We ran some quick tests on both **Bevy 0.12** and **Bevy 0.13** on the same machine on four complex scenes: [Bistro], [Sponza], [San Miguel] and [Hidden Valley].
+
+![A high polygon, realistically lit screenshot of a beautiful cafe with a tree in the background.](San_Miguel_13.jpg)
+
+As you can see, these scenes are substantially more detailed than most video game environments, but that screenshot was being rendered in Bevy at better than 60 FPS at 1440p resolution!
+Between Bevy 0.12 and Bevy 0.13 we saw frame times decrease by about 5-10% across the scenes tested. Nice work!
+
+![A graph showing the FPS of the different scenes. Bistro went from 90 FPS to 120 FPS, while the other scenes improved slightly. All scenes were 60-120 FPS.](rendering-perf-graph.svg)
+
+[Bistro]: https://github.com/DGriffin91/bevy_bistro_scene
+[Sponza]: https://github.com/DGriffin91/bevy_sponza_scene
+[San Miguel]: https://github.com/DGriffin91/bevy_san_miguel_scene
+[Hidden Valley]: https://blog.polyhaven.com/hidden-alley/
+
+## Unload Rendering Assets from RAM
+
+<div class="release-feature-authors">authors: @JMS55, @mockersf, @brianreavis</div>
+
+Meshes and the textures used to define their materials take up a ton of memory:
+in many games, memory usage is the biggest limitation on the resolution and polygon count of the game!
+Moreover, actually transferring that data from system RAM (used by the CPU) to the VRAM (used by the GPU) can be a real performance bottleneck.
+
+**Bevy 0.13** adds the ability to unload this data from system RAM, once it has been succesfully transferred to VRAM. However, unloading the data from the CPU [can result in bugs], and make it harder to actually inspect or modify that data from your other game systems.
```

Comment:
[can result in bugs]

Broken link? Also, this section is kind of unclear. Result in bugs is very vague. The actual problem is users or libraries like rapier depending on mesh data in a system in order to generate colliders and stuff at runtime, which need the asset data in memory until the collider is generate, and then it can be unloaded. 

Also this section could be a bit more clear, the reason you'd want to unload render assets is to save on RAM since you only need it in VRAM for the renderer, and we strongly recommend users to set the persistence policy to render only.

Reply:
This was intended to link to the revert PR.

Reply:
Adressing this now

Reply:
Link isn't actually broken. It just renders as broken in vscode's syntax highlighting